app/controllers/request-psp-test-account/post.controller.js

```diff
@@ -1,32 +1,23 @@
 'use strict'
 
 const { response } = require('../../utils/response')
-const zendeskClient = require('../../services/clients/zendesk.client')
 const getAdminUsersClient = require('../../services/clients/adminusers.client')
+const publicAuthClient = require('../../services/clients/public-auth.client')
+const { ConnectorClient } = require('../../services/clients/connector.client')
 const logger = require('../../utils/logger')(__filename)
 const { CREATED, NOT_STARTED, REQUEST_SUBMITTED } = require('../../models/psp-test-account-stage')
 const adminUsersClient = getAdminUsersClient()
+const { CONNECTOR_URL } = process.env
+const connectorClient = new ConnectorClient(CONNECTOR_URL)
 
 async function submitRequestAndUpdatePspTestAccountStatus (req) {
-  const message = `Service name: ${req.service.name}
-    Service ID: ${req.service.externalId}
-    PSP: 'Stripe'
-    Email address: ${req.user.email}
-    Time: ${new Date().toISOString()}
-    Service created at: ${req.service.createdDate || '(service was created before we captured this date)'}`
-
-  const zendeskOpts = {
-    email: req.user.email,
-    name: req.user.email,
-    type: 'task',
-    subject: `Request for Stripe test account from service (${req.service.name})`,
-    tags: ['govuk_pay_support'],
-    message: message
-  }
-
-  await zendeskClient.createTicket(zendeskOpts)
-  await adminUsersClient.updatePspTestAccountStage(req.service.externalId, REQUEST_SUBMITTED)
-  logger.info('Request submitted for Stripe test account')
+  const ids = await connectorClient.requestStripeTestAccount(req.service.externalId)
+  logger.info(`Stripe connect account ${ids.stripe_connect_account_id} was created`)
+  logger.info(`Gateway account with id ${ids.gateway_account_id} and external id ${ids.gateway_account_external_id} was created in connector`)
+  logger.info(`Service id ${req.service.external_id}`)
```

Comment:
I'm wondering if it might be better to include all this in a single logging statement, e.g. "stripe test account created for service id xxxx with gateway account id xxx, gateway account external id xxx and stripe connect account xxxx" so that searching for one of ids in Splunk will bring up all the info?